FreeRTOS_TCP_WIN.c

```diff
@@ -460,7 +460,13 @@
                 {
                     /* Could call vListInitialiseItem here but all data has been
                     * nulled already.  Set the owner to a segment descriptor. */
+					
+					/*
+					* Without call of the vListInitialiseItem, the list aata integrity check bytes aren't set
```

Comment:
The integrity check bytes are meant for debugging while the code is still in development - I am unsure if someone would need this in the "production" code. The verbiage in the History.txt of FreeRTOS-Kernel is: "_Introduced configUSE_LIST_DATA_INTEGRITY_CHECK_BYTES which, when set, will include known values in both list and list item structures. The values are intended to assist debugging. If the values get overwritten then it is likely application code has written over RAM used by the kernel._"

If that is the case, you are free to make changes in the source as you please for testing. But in the production code - once you are sure that the application is free from bugs which might overwrite kernel RAM - the macro should be defined back to 0.

Reply:
On one side, you have right. When the code is released, this configUSE_LIST_DATA_INTEGRITY_CHECK_BYTES should be 0 and isn't used. But I whould help, for debugging and searching problems, when then Inegrity check bytes are always correct set, if the configuare is active. So, many people need not so long search, where the problems commes